who-metrics-ui/src/components/RepositoriesTable.tsx

```diff
@@ -455,6 +475,53 @@ const RepositoriesTable = () => {
         </a>
       ),
     },
+    Topics: {
+      key: 'topics',
+      name: 'Topics',
+      width: 275,
```

Comment:
I didn't know we could this! This would fix the issue with column selectors. Nice!

Reply:
Yeah, I found out by guessing 😅 It's a very nice feature.